pyproject.toml

```diff
@@ -112,9 +113,14 @@ known-first-party = [
 
 [tool.ruff.per-file-ignores]
 "__init__.py" = ["F401", "D104"]
-"**/test_*.py" = [
+"**test**" = [
```

Comment:
seems to be much more relaxed, what weren't we catching with **/test_*.py ?

Reply:
I changed to also capture `conftest.py`.

Reply:
Can we do this in Ruff,   "**/test_*.py", "conftest.py" = [ .... ] so we don't duplicate the exceptions?

Reply:
I did try something similar and now that. no luck. Will check the docs tho, maybe there's a way 🤔